tests/notification/test_validation.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from callisto.notification.models import EmailNotification
+
+
+class EmailValidationTest(TestCase):
+
+    def test_duplicate_emails_not_allowed(self):
```

Comment:
can we add a test that creates two duplicate-named emails on different sites, then tries to add the other site to one of them and verify that an error is also thrown then?